monkey/infection_monkey/exploit/log4shell_utils/ldap_server.py

```diff
@@ -202,20 +199,21 @@ def stop(self, timeout: Optional[float] = None):
                         terminates. If `timeout` is a positive floating point number, this method
                         blocks for at most `timeout` seconds.
         """
-        if self._server_process is None:
+        from twisted.internet import reactor
```

Comment:
Perhaps this should be wrapped in a reusable function called `import_twisted_reactor()` that can be called from `_run_twisted_reactor()` as well.

Reply:
It [doesn't seem easy to do that with a decorator](https://stackoverflow.com/questions/29018731/python-import-module-from-decorator).

Reply:
We can't do that because the import is only available in that function's scope then.

Reply:
Fair enough.